GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1522,24 +1522,13 @@ private void RebaseToolStripMenuItemClick(object sender, EventArgs e)
 
             if (revisions.Count == 2)
             {
-                string? to = null;
-                string? from = null;
-
                 string currentBranch = RevisionGrid.CurrentBranch.Value;
-                var currentCheckout = RevisionGrid.CurrentCheckout;
 
-                if (revisions[0].ObjectId == currentCheckout)
-                {
-                    from = revisions[1].ObjectId.ToShortString();
-                    to = currentBranch;
-                }
-                else if (revisions[1].ObjectId == currentCheckout)
-                {
-                    from = revisions[0].ObjectId.ToShortString();
-                    to = currentBranch;
-                }
+                string? to = currentBranch;
```

Comment:
A comment would be good here
```suggestion
                // Set defaults in form, rebase the range from first selected commit to HEAD on last selected
                string? to = currentBranch;
```

Reply:
I have added a comment. Have a look if it's good enough....